src/core/bau_inputs.jl

```diff
@@ -46,6 +46,12 @@ function BAUInputs(p::REoptInputs)
     cop = Dict(t => 0.0 for t in techs.cooling)
     thermal_cop = Dict{String, Float64}()
     production_factor = DenseAxisArray{Float64}(undef, techs.all, p.time_steps)
+    tech_renewable_energy_pct = DenseAxisArray([p.tech_renewable_energy_pct[t] for t in techs.all], techs.all)
+    # !!! note: tech_emissions_factors in lb / kWh (gets multiplied by kWh of fuel burned, not kWh electricity consumption, ergo the use of the HHV instead of fuel slope)
+    tech_emissions_factors_CO2 = DenseAxisArray([p.tech_emissions_factors_CO2[t] for t in techs.all], techs.all)
```

Comment:
same comment as previous about potentially indexing emissions factors on pollutant (and tech) rather than having unique variable names for each

Reply:
It's handled by the try/catches in the functions that call get_EASIUR2005(), i.e. easiur_costs() and easiur_escalation_rates(). False is kind of a weird thing to return when it fails though, so I changed them to nothing. I'm also changing the warnings to errors because that's what they are.